apps/staking/hooks/useStakes.tsx

```diff
@@ -16,13 +21,18 @@ import type { Address } from 'viem';
  * @returns The stakes and related data for the connected wallet.
  */
 export function useStakes(overrideAddress?: Address, autoUpdateIntervalOverride?: number) {
```

Comment:
Would be good to specific that this is in Seconds

Reply:
removed